Framework/PythonInterface/test/python/mantid/plots/datafunctionsTest.py

```diff
@@ -834,6 +835,16 @@ def test_errorbars_hidden_returns_true_on_container_with_invisible_connecting_li
         [bars.set_visible(False) for bars in container[2]]
         self.assertTrue(mantid.plots.datafunctions.errorbars_hidden(container))
 
+    def test_get_bin_indices_returns_a_range_with_no_monitors(self):
```

Comment:
@martyngigg this might be overkill??

Reply:
I think this is okay. We have 2 code paths so 2 tests is fine.